Description: replace ava by tape
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-10-12

--- a/test.js
+++ b/test.js
@@ -1,5 +1,5 @@
-import test from 'ava';
-import uniqueString from '.';
+const test = require('tape');
+const uniqueString = require('.');
 
 test('main', t => {
 	t.is(uniqueString().length, 32);
@@ -17,4 +17,5 @@
 
 		created.add(string);
 	}
+    t.end();
 });
